sitemedia/js/search.js

```diff
@@ -113,6 +113,8 @@ $(function(){
     }
 
     function onPageLoad() {
+        // remove no-js styles for CSS
```

Comment:
This makes sense, though maybe `remove no-js styles for CSS now that Javascript has been detected`?